vsc_pdf_template_transformer/lib/src/vsc_pdf_template_transformer_base.dart

```diff
@@ -13,17 +14,17 @@ class Transformer {
 
   static pw.Document buildPdf(
       Map<String, dynamic> template, Map<String, dynamic> data) {
-    final pdf = pw.Document();
+    final pdf = TplDocument.fromJson(template).toPdf();
+    final content = template['children'][0]['children'][0];
     pdf.addPage(pw.Page(
```

Comment:
Document and Page should be defined in the template JSON like other widgets. It seems we're missing TplPage.  We should just enumerate over TplDocument's children add add those as pages. Each child will be either TplPage or TplMultiPage. TplPage You will have to handle the `Page.build()` method in TplPage when evaluating it. TplPage will build a single widget. TplMultiPage will build multiple widgets. Unfortunately due to the design of Document and Page/MultiPage, you will have to handle them specially here.